specs/http-api/README.md

```diff
@@ -323,18 +367,39 @@ Uses DID authn via Bearer token in header.
 ## List Exchanges
 
 ### Description
-Returns an array of tbdex message arrays
-
-### Authentication
-Uses DID authn via Bearer token in header.
+Returns an array of tbdex message arrays (a list of exchanges)
 
 ### Endpoint
 `GET /exchanges`
 
+### Authentication
+A tbdex http client must send an Http Auth Header in the form of a JWT `Bearer` token.
+
+The JWT must consist of JWS header and JWT payload containing the below fields, and signed by the client's DID.
+
+#### JWS Header
```

Comment:
```suggestion
#### JOSE Header
```

Reply:
I'd prefer to leave this as JWS header, because from what I'm reading, JOSE can include many kinds of headers like JWS, JWE, JWK, and JWA.

But, to describe the request token header, I want to specify that it must be a `JWS` header, which must include `alg`, `typ`, and `kid` fields.